go.work

```diff
@@ -0,0 +1,3 @@
+go 1.21.4
```

Comment:
I removed the patch so not everyone has to use the exact same version. I also added the other go modules to the workspace so we can remove require statements.